build.gradle

```diff
@@ -20,6 +23,8 @@ buildscript {
         classpath 'org.ajoberstar:grgit:1.5.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 
+        classpath "org.mozilla.android:gradle-plugin:0.1"
```

Comment:
A comment linking to the plugin repo would be good.  As is, it's pretty unclear what this is doing.